### DISCUSSION TITLE
Feedback pls

### DISCUSSION THREAD

**Discussion by not-nef** [Futura-Py/futura-py.github.io#1] [General]
Please tell me what you think or what you want changed before i move on to adding the projects

**Comment by not-nef** [comment]
@sumeshir26 @im-coder-lg @Moosems

**Reply by Moosems** [reply]
Try making a JS script that fetches people from [here](https://github.com/orgs/Futura-Py/people) or a GH api so it is dynamic :D. Nice look though! I'd round the edges of the main picture and make a little bound box around the about div. I'd also put the about and members divs into a master div so you can modularly add divs going downward.

**Reply by not-nef** [reply]
> Try making a JS script that fetches people from [here](https://github.com/orgs/Futura-Py/people) or a GH api so it is dynamic :D. 

Thats a damn good idea, im gonna do that and the other stuff as soon as im finished with the weather pr

**Comment by sumeshir26** [comment]
Should we also add pages for our different projects?
Also should I get rid of timerx-app.netlify.app ?

**Reply by Moosems** [reply]
Make a navbar too for the home page and contents page.

**Reply by not-nef** [reply]
> Should we also add pages for our different projects? Also should I get rid of timerx-app.netlify.app ?
you can get rid of it when this website is finished

**Reply by not-nef** [reply]
> Make a navbar too for the home page and contents page.

i will later

**Reply by Moosems** [reply]
@not-nef Can you make an svg of the logo?

**Reply by not-nef** [reply]
@sumeshir26 has the svg

**Reply by sumeshir26** [reply]
@Moosems I have lost the original source file of the logo, hence cannot provide a SVG unfortunately :( I will try some online converters and try to get a workable SVG.

**Reply by sumeshir26** [reply]
![97777018 (4)](https://github.com/Futura-Py/futura-py.github.io/assets/68823982/c26b559d-0609-43de-bd9d-957ec16899ab)
 got it

**Reply by Moosems** [reply]
The shadow looks kinda odd ngl.

**Reply by sumeshir26** [reply]
yeah, but no idea on how to fix it

**Reply by lg-nineoneninethreenine** [reply]
Hmm, yeah. We'll add it here, as a branched website or something.

**Reply by Moosems** [reply]
I already started it by just adding a folder for the the project pages.

**Comment by not-nef** [comment]
everyone that wants their name listed on the website has to publicize their org membership at https://github.com/orgs/Futura-Py/people or the api wont pick them up

**Reply by sumeshir26** [reply]
Done
